content/blog/productivity-porn.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Productivity Porn"
+date: 2022-06-28
+draft: false
+---
+
+I have a pornography problem, but it's not what you're thinking. In fact if you're anything like me you might too.
+
+It is well understood that social media can have a negative impact on your life. Whether you are doom-scrolling on Instagram, furiously retweeting on Twitter, or on your fifth hour of Youtube, you might experience:
+
+- **Feelings of jealousy, anxiousness, and loneliness** induced by only seeing the heavily curated feeds of those who you follow. Ultimately, leading to unrealistic expectations about what your life should be like mixed with a constant bombardment of FOMO.
+- **An increasingly polarized, misinformed, and enraged outlook on life** as the incentives of our digital world push you deeper into a filter bubble. In the extreme case even leading to radicalization.
+- **A crippling loss of time and agency** caused by the addictive and distracting nature of these platforms. The push notification acting as a phantasmal chain tethering us to our phone. Watching content not because you truly want to but just --- because?
+
+But, what I've recently come to realize is that there is an even more pernicious way in which social media is negatively impacting my life. Social media has become my productivity porn.
+
+Productivity porn is anything that after having been consumed makes you feel like you were productive when in reality you didn't actually do anything. Just like regular pornography it stimulates you without even performing the act. It is a hollow imitation of the real thing.
+
+Examples of productivity porn include but are not limited to: reading a tweet by a top VC about how to become a better startup founder; watching a Youtube video about the 7 mistakes you need to avoid at the gym; perusing a Hacker News thread about how to improve the code you write. All of these activities deceptively make you feel like you've done something productive. "I just learned something new", you tell yourself. And well this is true, you never actually did the thing you were setting out to do in the first place. A viscious cycle forms where you spend all your time thinking about doing things instead of actually doing them. As this cycle continues your productivity approaches zero.
+
+Keep in mind that productivity is a personal thing. It can look like a variety of things: founding a startup, working out at the gym, building a side project, reading a book, growing a garden. Regardless, the key is that you benefit from doing the actual activity, but not from thinking about doing it. Hence the problem with productivity porn.

```

Comment:
```suggestion
Keep in mind that productivity is a personal thing. It can look like a variety of things: founding a startup, working out at the gym, building a side project, reading a book, growing a garden. Regardless, the key is that you benefit from doing the actual activity (not from thinking about doing it). Hence the problem with productivity porn.

```
Mixing it up a bit to throw in some parentheses instead of more commas.